dist/debian/gdb/control.mustache

```diff
@@ -0,0 +1,16 @@
+Source: {{product}}-gdb
+Maintainer: Takuya ASADA <syuu@scylladb.com>
+Homepage: http://scylladb.com
+Section: devel
+Priority: optional
+X-Python3-Version: >= 3.4
+Standards-Version: 3.9.5
+
+Package: {{product}}-gdb
+Architecture: amd64
+Description: A standalone gdb that can be moved around different Linux machines
+ This is a self-contained gdb that can be moved around
+ different Linux machines as long as they run a new enough kernel (where
+ new enough is defined by whichever Python module uses any kernel
```

Comment:
obviously this has nothing to do with python, so this is just a copy&paste error